### PR TITLE
Expended menu bar with javascript

### PR DESCRIPTION
**Pull request by Elijahdre**
# Hamburger menu with JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,13 @@
   <header>
     <nav>
       <label class="logo">Sahar</label>
-      <span><img src="img/Enabled.png" alt="burger" class="burger"></span>
-      <ul>
+      <ul id="expend" class="sidebar">
+        <li><a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a></li>
         <li><a href="#">Portfolio</a></li>
         <li><a href="#">About</a></li>
         <li><a href="#">Contact</a></li>
```
Comment (ShadyShawkat) [inline index.html]: - [x] Kindly make sure that when the user clicks on the nav links in the mobile menu and in the desktop navigation navbar, the page scrolls to the appropriate section of the page (when the user clicks on "portfolio" in the mobile menu or the desktop navigation bar, the page scrolls to the portfolio section). You can use the `href` property to do so.

- [x] Currently, when the user opens the mobile menu on mobile view, the nav links on desktop view are hidden, and they stay hidden even after the user closes the mobile menu. (Please fix this behavior)

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -10,12 +10,13 @@
...
-      <span><img src="img/Enabled.png" alt="burger" class="burger"></span>
-      <ul>
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
+      <ul class="sidebar" id="expend">
+        <li><a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a></li>
+        <li><a href="#portfolio" onclick="closeNav()">Portfolio</a></li>
+        <li><a href="#about" onclick="closeNav()">About</a></li>
+        <li><a href="#contact" onclick="closeNav()">Contact</a></li>
```
Comment (prolajumokeoni) [inline index.html:17]: #### OPTIONAL

- [ ] Kindly increase the font size of each of the mobile nav links

**Review by prolajumokeoni — APPROVED** [review]
Hi @Ream

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**